integration-tests/chopsticks/src/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { Keyring } from '@polkadot/keyring'
+
+const keyring = new Keyring({ type: 'ed25519', ss58Format: 38 })
+
+export const keysAlice = keyring.addFromUri('//alice', undefined, 'ed25519')
+export const keysBob = keyring.addFromUri('//bob', undefined, 'ed25519')
+export const keysCharlie = keyring.addFromUri('//charlie', undefined, 'ed25519')
+
+export function toNumber(value: string | undefined): number | undefined {
+	if (value === undefined) {
+		return undefined
+	}
+
+	return Number(value)
+}
+
+export const KILT = BigInt(10 * 10e15)
+export const DOT = BigInt(10 * 10e10)
```

Comment:
Are you sure these are correct? This to me sounds like 100 DOTs, not one DOT. Did you want to refer to a single DOT?

Reply:
Good catch: [0ebcde8](https://github.com/KILTprotocol/kilt-node/pull/614/commits/0ebcde8b490b1a24a07d4825f09acf7035cdf810)